mysql_merge/mysql_merger.py

```diff
@@ -329,35 +320,121 @@ def copy_data_to_target(self):
+            for pk in self._db_map[table_name]['primary'].keys():
+                on += "" if len(on) == 0 else " AND"
+                on += "`%(source_db)s`.`%(table)s`.`%(pk)s` = `%(destination_db)s`.`%(table)s`.`%(pk)s`" % {"source_db": self._source_db['db'],
+                                                                                                            "destination_db": self._destination_db['db'],
+                                                                                                            "table": table_name,
+                                                                                                            "pk": pk}
+                where += "" if len(where) == 0 else " OR"
+                where += "`%(destination_db)s`.`%(table)s`.`%(pk)s` IS NULL" % {"destination_db": self._destination_db['db'],
+                                                                                "table": table_name,
+                                                                                "pk": pk}
+
+            for column in columns:
+                where += " OR `%(source_db)s`.`%(table)s`.`%(col)s` <> `%(destination_db)s`.`%(table)s`.`%(col)s`" % {"source_db": self._source_db['db'],
+                                                                                                                      "destination_db": self._destination_db['db'],
+                                                                                                                      "table": table_name,
+                                                                                                                      "col": column}
+
+            query = "SELECT 1 FROM `%(source_db)s`.`%(table)s` LEFT JOIN `%(destination_db)s`.`%(table)s` ON %(on)s WHERE  %(where)s" % {
+                'destination_db': self._destination_db['db'],
+                'source_db': self._source_db['db'],
+                'table': table_name,
+                'where': where,
+                'on': on
+            }
+
+            try:
+                rows = self._cursor.execute(query)
+                while True:
+                    row = rows.fetchone()
+                    if not row:
+                        break
+                    raise Exception(
+                        "Source enum Table %s has values incompatible with destination database and will be merged" % table_name)
+
+            except Exception, e:
+                handle_exception("Exception occured while comparing '%(t)s' enum table" % {
+                                 "t": table_name}, e, self._conn)
+
+            self._logger.log(
+                "----> Enum Table %s is compatible with destination database and will be ignored during merge" % table_name)
+            self._db_map.pop(table_name)
+            self._destination_db_map.pop(table_name)
+
+    def get_increment_value(self, table_name):
+        if table_name in self._config.increment_step:
```

Comment:
:D (ignore me)

`increment_step = self._config.increment_step[table_name if table_name in self._config.increment_step else 'default']`